src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.ConfigUpdaters/ConnectionStringsConfigUpdater.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml.XPath;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.ConfigUpdaters
+{
+    public class ConnectionStringsConfigUpdater : IUpdater<ConfigFile>
```

Comment:
This looks like a lot of duplicated code from the appsetting one. Can you create a shared base class or something? Or combine it into that one?

Reply:
I've moved the duplicated code into a shared base class. Let me know what you think of this.

Reply:
Great, thanks!